test/validation/api/ipsec/ipsec.c

```diff
@@ -351,6 +351,18 @@ int ipsec_check_esp_aes_gcm_256(void)
 				ODP_AUTH_ALG_AES_GCM, 0);
 }
 
+int ipsec_check_test_sa_update_seq_num(void)
```

Comment:
This is a very minor thing, but it would be good to not place this function in the middle of the algorithm capability check functions but before or after them. It would keep the file slightly more easy to read as a whole.

Reply:
Agreed. Moved the function after the algorithm check capability functions.